portfolio/src/main/webapp/script.js

```diff
@@ -28,28 +29,47 @@ async function getPhrase(){
         commentSection.appendChild(addName);  
         commentSection.appendChild(addMessage);  
         document.getElementById("section").appendChild(commentSection);
-    }    
+    }  
+  
 }
 
 async function removePhrase(){
     let response = await fetch('/delete-data',  {method:'post'});
     getPhrase();
 }
 
-function loadChart(){
+async function loadChart(){
+    let commentMap = new Map();
+    let response = await fetch('/load-data');
+    let list = await response.json();
     let stats = new google.visualization.DataTable();
-      stats.addColumn('string', 'Action');
-      stats.addColumn('number', 'Percentage');
-      stats.addRows([
-        ['Programming', 0.65],
-        ['Biking', 0.10],
-        ['Gaming', 0.25]
-      ]);
-
-      // Instantiate and draw the chart.
-      var chart = new google.visualization.PieChart(document.getElementById('activityPieChart'));
-      chart.draw(stats, null);
+    
+    stats.addColumn('string', 'Name');
+    stats.addColumn('number', 'Total Comments');
+
+    countComments(commentMap, list);
+
+    for(let [name, numComments] of commentMap){      
+        stats.addRows([
+            [name, numComments],
+        ]);      
+    }
+
+    // Instantiate and draw the chart.
+    var chart = new google.visualization.BarChart(document.getElementById('commentBarChart'));
+    chart.draw(stats, null);
+}
+
+function countComments(map, list){  
+    for(const element of list){
+        let name = element.name;
+
+        if(map.has(name))
+            map.set(name, map.get(name) + 1); //add one to current value
+        else
+            map.set(name, 1); //first occurence of that comment
```

Comment:
Make sure to use braces (https://google.github.io/styleguide/jsguide.html#formatting-braces-all)

Reply:
Got it, I'll add braces from now on. I'll push a new commit with the change